pipelines/cellpainting/cpd_max_projection_pipeline.wdl

```diff
@@ -15,23 +15,28 @@ workflow cpd_max_projection_distributed {
     # Specify input file information, images directory & extension
     String images_directory_gsurl
     String? file_extension = ".tiff"
+    String load_data_directory_gsurl
```

Comment:
I wonder what you think about making this a specific file path like
```wdl
String load_data_gsurl
```

Reply:
I'm not sure, maybe I do mean `File`.  I kind of thought a `File` was an input type that could only be used for a `task` and not a `workflow`, but I'm not sure if I made that up.  Since `File`s get localized to the VM, I assumed that it was only applicable for a `task`, since a `workflow` has no associated VM

Reply:
Clearly I am not right about this though, because there are plenty of examples of `workflow`s that have `File` inputs.  I'm kind of confused about what it means... I guess there's really no difference, at the level of a `workflow`?

Reply:
Actually, I think maybe the string manipulation
```wdl
sub(load_data_gsurl, "/+$", "")
```
will only work if `load_data_gsurl` is a `String` and not a `File`.  I could be wrong though!

Reply:
(This is why I want to get the automatic testing framework set up  :)  )

Reply:
adding as inputs:
```
    File load_data
    File load_data_with_illum
```